.github/workflows/flowzone.yml

```diff
@@ -533,8 +559,6 @@ jobs:
         run: |
           echo "not yet implemented!"
 
-      # TODO: publish to npmjs
```

Comment:
On merge we still need to re-publish with the final version, right?

Reply:
@klutchell we'll need to do something similar with `npm dist-tags ...` as [resinci-deploy](https://github.com/product-os/resinci-deploy/blob/master/lib/npm.ts)